backend/import/xml/parser.js

```diff
@@ -116,6 +117,15 @@ const getTrueType = (type) => (
   type.split("-")[0].trim()
 );
 
+const manaSymbolRegExp = /(?:(?<!\/)(?:\d+|[^/\s])(?!\/))|\S\/\S|\/\//g;
+const addManaCostBrackets = (manacost) => {
+  if (manaSymbolRegExp.test(manacost)){
+    let manaSymbols = manacost.match(manaSymbolRegExp);
+    return manaSymbols.map((manaSymbol) =>{
+      return manaSymbol === "//" ? ` ${manaSymbol} ` : `{${manaSymbol}}`;}).join("");
+  } else return manacost;
+};
```

Comment:
_style_  (totally my preference, please feel free to ignore)

I would write that function with an "early return to reduce nesting, and use indenting to show the chained functions
```js
const addManaCostBrackets = (manacost) => {
  if (!manaSymbolRegExp.test(manacost)) return manacost

  return manacost
    .match(manaSymbolRegExp)
    .map((manaSymbol) => manaSymbol === "//" ? ' // ' : `{${manaSymbol}}`)
    .join("");
};
```

Reply:
Oh, that's awesome, thanks! Much clearer to read too!